ui/page/wallet/wallet_settings_page.go

```diff
@@ -373,26 +430,25 @@ func (pg *SettingsPage) changeSpendingPasswordModal() {
 			}
 			pg.wallet.LockWallet()
 
-			// change password
-			newSpendingPasswordModal := modal.NewCreatePasswordModal(pg.Load).
-				Title(values.String(values.StrChangeSpendingPass)).
-				EnableName(false).
-				PasswordHint(values.String(values.StrNewSpendingPassword)).
-				ConfirmPasswordHint(values.String(values.StrConfirmNewSpendingPassword)).
-				SetPositiveButtonCallback(func(walletName, newPassword string, m *modal.CreatePasswordModal) bool {
-					err := pg.wallet.ChangePrivatePassphraseForWallet(password,
-						newPassword, sharedW.PassphraseTypePass)
+			currentPassword = password
+			if pg.AssetsManager.DexcReady() {
+				// Check if this wallet exists in the dex client.
```

Comment:
what does it mean that a "wallet exists in the dex client"?
A more correct expression would be "Check if this wallet is used by the dex client".